### ISSUE TITLE
Add slippage param when supplying liquidity

### ISSUE THREAD

**Issue by hetfly** [balancednetwork/balanced-network-interface#1317]

**Comment by FezBox** [comment]
@hetfly is this still a valid issue?

**Comment by hetfly** [comment]
@FezBox yes, but not a priority imo. There was a new feature added while providing liquidity to protect users. Now it defaults to 2%, in the future we could add an UI input to edit the default slippage.

**Comment by FezBox** [comment]
when providing liquidity we want t he user to be able to specify maximum slippage

**Comment by parrot9design** [comment]
Here's how I thought we could handle this:

<img width="410" alt="lp-slippage" src="https://github.com/user-attachments/assets/67594cd6-9afa-460d-91eb-3f59fd9c31d5">

(Makes it easy to add in the transfer fee(s) once we take this cross-chain)

The slippage tooltip should say:
>If the price slips by more than this, your liquidity transaction will fail.
